x-pack/plugins/alerting/server/rules_client/methods/bulk_enable.ts

```diff
@@ -183,114 +189,120 @@ const bulkEnableRulesWithOCC = async (
         });
       }
 
-      await pMap(rulesFinderRules, async (rule) => {
-        try {
-          if (scheduleValidationError) {
-            throw Error(scheduleValidationError);
-          }
-          if (rule.attributes.actions.length) {
-            try {
-              await context.actionsAuthorization.ensureAuthorized({ operation: 'execute' });
-            } catch (error) {
-              throw Error(`Rule not authorized for bulk enable - ${error.message}`);
+      await pMap(
+        rulesFinderRules,
+        async (rule) => {
```

Comment:
can we inline this so it doesn't create such a big diff

Reply:
Inline what?

Reply:
```
await pMap(rulesFinderRules, async (rule) => {
        try {
          if (scheduleValidationError) {
...
```

Reply:
sounds good